dynamodb/dynamodb-types.w

```diff
@@ -179,6 +192,9 @@ pub struct TableProps {
   timeToLiveAttribute: str?;
   globalSecondaryIndex: Array<GlobalSecondaryIndex>?;
   pointInTimeRecovery: bool?;
+  billingMode: BillingMode?;
+  removalPolicy: RemovalPolicy?;
+  deletionProtection: bool?;
```

Comment:
I'm thinking maybe we just keep `deletionProtection` for now since RemovalPolicy doesn't really have a Terraform equivalent AFAIK
```suggestion
  billingMode: BillingMode?;
  deletionProtection: bool?;
```

Reply:
I have removed it as per suggestion. I still feel that needs to be there for anyone that would normally set those options in AWS CDK.